packages/e2e-tests/helpers/elements-panel.ts

```diff
@@ -293,7 +297,216 @@ export async function toggleMarkupNode(page: Page, locator: Locator, open: boole
   }
 }
 
-export async function waitForSelectedElementsRow(page: Page, text: string): Promise<void> {
+export async function waitForSelectedElementsRow(page: Page, text: string): Promise<Locator> {
   const locator = await getElementsRowWithText(page, text, true);
   await locator.waitFor();
+  return locator;
+}
+
+export async function typeKeyAndVerifySelectedElement(
+  page: Page,
+  key: string,
+  expectedElement: string
+) {
+  debugPrint(page, `Typing ${key}...`);
+  await page.keyboard.press(key);
+  await waitForSelectedElementsRow(page, expectedElement);
+}
+
+// `doc_stacking.html` data and utils
```

Comment:
organizational nit: In our screenshot tests, we put helpers that are specific to a group of tests in a folder with those tests. We don't have to do that here but it does seem a little odd to have test-specific helpers in the general "elements-panel" utils file.

We don't have the same folder organization with these e2e tests so that's not as feasible, but maybe we could at least move these into a `helpers/stack-test-case.ts` file so it's a bit clearer?

Alternately you could just move the `StackingTestCase` and `stackingTestCases` data out somewhere (`data/stacking-test-cases.ts`?) and leave the methods below in this file.